### PR TITLE
doc: review edits for config tool doc

### PR DESCRIPTION
**Pull request by dbkinder**
Update material from PR #3559 with review and clarity edits.

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

File: doc/tutorials/acrn_configuration_tool.rst [hunk 1]
```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
...
-      the tool will do sanity check on the input based on ``$(BOARD).xml`` and then generate
-      customized scenario based VM configurations in ``$(SCENARIO).xml``;
-   #. In the UI of configure tool, continue to input launch script parameter for
-      post-launched User VM. The tool will check the input based on both ``$(BOARD).xml``
-      and ``$(SCENARIO).xml`` and then generate launch script based VM configurations in
-      ``$(LAUNCH_PARAM).xml``;
+      The tool in this step is still under development. Until then,
+      you can input settings by editing the target XML file manually.
+
+   a. Copy ``$(BOARD).xml`` to the host develop machine.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst]: @lirui34 Can you provide specific commands to do this

File: doc/tutorials/acrn_configuration_tool.rst [hunk 2]
```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
...
-   #. **[TO BE DEVELOPED]** Run ``python3 launch_cfg_gen.py --board $(BOARD).xml
-      --scenario $(SCENARIO).xml --launch $(LAUNCH_PARAM).xml$`` under ``misc/launch_config``
-      folder, it will generate launch script for the specified post-launch User VM;
+   Python tools are used to generate configurations in patch format.
+   The patches will be applied to your local ``acrn-hypervisor`` git tree
+   automatically.
+
+   a. Generate a patch for the board-related configuration with::
+
+         cd misc/board_config
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst]: @lirui34 Does this folder already exist, or must the user create it first?  (Same comment for the other two cases below).  Where is this misc folder?

File: doc/tutorials/acrn_configuration_tool.rst [hunk 3]
```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
...
-   a. Copy ``$(BOARD).xml`` to host develop machine;
-   #. Run a UI based configure tool on host machine to input the expected scenario settings,
-      the tool will do sanity check on the input based on ``$(BOARD).xml`` and then generate
-      customized scenario based VM configurations in ``$(SCENARIO).xml``;
-   #. In the UI of configure tool, continue to input launch script parameter for
-      post-launched User VM. The tool will check the input based on both ``$(BOARD).xml``
-      and ``$(SCENARIO).xml`` and then generate launch script based VM configurations in
-      ``$(LAUNCH_PARAM).xml``;
+      The tool in this step is still under development. Until then,
+      you can input settings by editing the target XML file manually.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst]: @lirui34 Where does the user learn what and how to edit this XML file?

File: doc/tutorials/acrn_configuration_tool.rst [hunk 4]
```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
...
 
    acrn-hypervisor/misc/acrn-config/
 
-Below is offline configure tool working flow:
+Here is the offline configuration tool workflow:
 
 #. Get board info.
 
-   a. Setup native Linux environment on target board.
+   a. Set up native Linux environment on target board.
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst]: What does the user need to do to set up this environment?

File: doc/tutorials/acrn_configuration_tool.rst [hunk 5]
```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
...
    #. Copy ``target`` folder into target file system and then run
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst]: @lirui34 Can you provide specific commands to do this

File: doc/tutorials/acrn_configuration_tool.rst [hunk 6]
```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
...
+      will be generated in the ``./out/`` folder. (Here $(BOARD) is the
+      specified board name)
 
-       | **Native Linux requirement:**
-       | **Release:** Ubuntu 18.04+ or ClearLinux 30210+
-       | **Tools:** cpuid, rdmsr, lspci, dmidecode(optional)
-       | **Kernel cmdline:** "idle=nomwait intel_idle.max_cstate=0 intel_pstate=disable"
+      | **Native Linux requirement:**
+      | **Release:** Ubuntu 18.04+ or Clear Linux 30210+
+      | **Tools:** cpuid, rdmsr, lspci, dmidecode(optional)
```
Comment (gvancuts) [inline doc/tutorials/acrn_configuration_tool.rst]:
```suggestion
      | **Tools:** cpuid, rdmsr, lspci, dmidecode (optional)
```

File: doc/tutorials/acrn_configuration_tool.rst [hunk 7]
```diff
@@ -5,130 +5,148 @@ ACRN Configuration Tool
...
 #. Get board info.
 
-   a. Setup native Linux environment on target board.
+   a. Set up native Linux environment on target board.
    #. Copy ``target`` folder into target file system and then run
       ``sudo python3 board_parser.py $(BOARD)`` command.
-   #. A $(BOARD).xml which includes all needed hardware specific information
-      will be generated at ``./out/`` folder. (Here $(BOARD) is the specified board name)
+   #. A $(BOARD).xml that includes all needed hardware-specific information
+      will be generated in the ``./out/`` folder. (Here $(BOARD) is the
```
Comment (gvancuts) [inline doc/tutorials/acrn_configuration_tool.rst]:
```suggestion
      will be generated in the ``./out/`` folder. (Here ``$(BOARD)`` is the
```

**Pull request merged**